src/resources/js/Pages/Dashboard.vue

```diff
@@ -28,6 +28,12 @@
                     <div>Updates in the last week: {{ lastWeek }}</div>
                     <div class="italic text-sm">Note: The above stats include <strong>only</strong> manual updates</div>
                 </div>
+
+                <div class="my-6">
+                    <h2 class="underline text-lg">Sites Updated Per Day</h2>
```

Comment:
I think I'd suggest "Sites Updated Since" rather than "Sites Updated Per Day" because looking at each day's data doesn't show how many sites were updated *on* that day. Instead, it shows how many sites haven't been updated *since* that day.

Reply:
I think that it's actually the former. But, honestly, I'm not sure. Here's how I interpret the query:

1. For all the availability updates newer than 30 days, 
2. group them by location [you end up with a set of updates per location]
2. Find the most recent update for each site [you end up with a single time per location]
3. Subtract each time from today [you end up with a single time per site, again, that is semantically the time since the location was most recently updated]
3. Round that time to the nearest 24 hour mark [you end up with a single time per location rounded to the nearest 24 hours whose semantic value is how many days ago the location was most recently updated]
4. group the times that match [you end up with groups whose size are semantically the number of locations updated most recently that many days ago]
5. count the size of each group. 

As a result, I _believe_ that you end up with the number of locations updated that many days since today. 

Does that make sense? Do you agree?

Reply:
That all makes sense, and I see how "Sites Updated Since" sounds like it might imply cumulative numbers. Maybe adding "last" would help so the title is "Sites Last Updated Per Day". When I think about "Sites Updated Per Day," it seems like that would need to include a count of all sites that had at least one update on that day, so the numbers would always be around that 350-400 mark for each day. 

Or we can leave it as is. I went ahead and deployed this after my comments because it's great data!